toolkit/docs/formats/imageconfig.md

```diff
@@ -182,6 +182,25 @@ A sample PackageLists entry pointing to three files containing package lists:
     "packagelists/cloud-init-packages.json"
 ],
 ```
+### PreInstallScripts
+
+There are customer requests that would like to use a kickstart file to install Mariner OS. Kickstart installation normally includes preinstall scripts that run before installation begins and are normally used to handle tasks like network configuration, determining partition schema etc. The preinstall script will populate the partition commands inside another file, usually /tmp/part-include. And here's an example of the partition command generated by kickstart: part / --fstype=ext4 --size=10240 --ondisk=/dev/sda. Refer to this link for more detailed information: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/installation_guide/sect-kickstart-syntax
```

Comment:
```suggestion
There are customer requests that would like to use a Kickstart file to install Mariner OS. Kickstart installation normally includes pre-install scripts that run before installation begins and are normally used to handle tasks like network configuration, determining partition schema etc. The `PreInstallScripts` field allows for running customs scripts for similar purposes. Sample Kickstart pre-install script [here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/5/html/installation_guide/s1-kickstart2-preinstallconfig).

Currently, Mariner's pre-install scripts are mostly intended to provide support for partitioning schema configuration. For this purpose, make sure the script creates a proper configuration file (example [here](https://www.golinuxhub.com/2018/05/sample-kickstart-partition-example-raid/)) under`/tmp/part-include`.
```

If I understood you correctly, I suggest these changes to better reflect what we currently support. Let me know, if this is accurate or not.